src/handlers/motions/motionStaked/motionStaked.ts

```diff
@@ -14,6 +15,8 @@ import {
 export default async (event: ContractEvent): Promise<void> => {
   const {
     contractAddress: colonyAddress,
+    logIndex,
```

Comment:
If these are the only two components of the key, we could just declare const at the top here: 
``` js
const messageKey = `${transactionHash} ${logIndex}`
```js
And then include that on every message instead of both the tx hash and log index

Then the front end doesn't need to know anything about them, it just takes a key and use it as is.

Reply:
And while I'm thinking about it, the order of the messages isn't going to change, right?

Therefore we don't really need the logIndex either. You could just use the item's array index when mapping through in the front end. This is [not recommended](https://react.dev/learn/rendering-lists) if the ordering might change, but since ours won't, would be one less thing to think about in the block-ingestor.